docs/development/variables/environment-variables.mdx

```diff
@@ -49,50 +49,51 @@ the `MAGE_REQUESTS_BASE_PATH` and `MAGE_ROUTES_BASE_PATH` environment variables
-| `REQUIRE_USER_AUTHENTICATION`   | Enable user authentication in Mage. [More information](/production/authentication/overview)                                                                     | 1                                                                        |
-| `SERVER_VERBOSITY`              | [More information](/development/observability/logging#server-logging)                                                                                           | See link                                                                 |
-| `SHELL_COMMAND`                 | Set shell command to use for the Mage terminal. Default command is `bash` for macOS/Unix and `cmd` for Windows.                                                 | `bash`, `cmd`, ...                                                       |
-| `ULIMIT_NO_FILE`                | Override the maximum number of open files allowed in Mage processes.                                                                                            | 8192                                                                     |
-| `USER_CODE_PATH`                | Set this path to be `/home/src/{your_project_name}` so that Mage starts your project and installs the packages in requirements.txt when starting the container. | `/home/src/mage_project`                                                 |
+| Variable name                    | Description                                                                                                                                                     | Example values                                                           |
+| -------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| `ACTIVE_DIRECTORY_DIRECTORY_ID`  | Set this value to your organization's tenant id to enable signing in with Microsoft account.                                                                    | [More info](/production/authentication/microsoft)                        |
+| `AUTHENTICATION_MODE`            | Set the authentication mode in Mage. Omitting the variable will default to basic password authentication.                                                       | `ldap`                                                                   |
+| `DATAFRAME_SAMPLE_COUNT_PREVIEW` | Number of lines to display in block sample output preview for dataframes. Not suggested to go over 10 lines (defaults to 10).                                   | 5                                                                        |
```

Comment:
Added this line to the docs to describe the `DATAFRAME_SAMPLE_COUNT_PREVIEW` env var. There were no other changes to the docs. The rest of the diff in this file was just formatting.